docs/testplans/study-viewer-HEAL/study-viewer-HEAL.md

```diff
@@ -0,0 +1,96 @@
+#HEAL PROJECT Study Viewer: Tiered Access Support
```

Comment:
I think you need to add spaces after the pound signs in each header title so that the markdown is formatted correctly. (see the preview mode -- these headers aren't rendering as intended)